source/install/self-managed-changelog.md

```diff
@@ -169,6 +169,7 @@ Latest Mattermost Releases:
  - Fixed an issue where the undo history was erased when links, tables, or code was pasted into the textbox.
  - Fixed an issue where Elasticsearch didn't properly start on startup when enabled. Also added a missing ``IsEnabled`` method to Elasticsearch.
  - Fixed an issue where text couldn't be copied from the post textbox.
+ - Fixed an issue where using **shift+tab** with a screen reader placed the cursor focus at the bottom of the channel rather than at the post that was linked to.
```

Comment:
```suggestion
 - Fixed an issue where using **SHIFT+TAB** with a screen reader placed the cursor focus at the bottom of the channel rather than at the post that was linked to.
```